packaging/templates/mainnet-v1/without-sentry/bor/pbss_config.toml

```diff
@@ -0,0 +1,179 @@
+# NOTE: Update and uncomment: `keystore`, `password`, and `unlock` fields.
+
+chain = "mainnet"
+# identity = "node_name"
+# verbosity = 3
+# vmdebug = false
+datadir = "/var/lib/bor/data"
+# ancient = ""
+db.engine = "pebble"
+state.scheme = "path"
+# keystore = "$BOR_DIR/keystore"
+# "rpc.batchlimit" = 100
+# "rpc.returndatalimit" = 100000
+syncmode = "full"
+# gcmode = "full"
+# snapshot = true
+# ethstats = ""
+# devfakeauthor = false
+
+# ["eth.requiredblocks"]
+
+# [log]
+    # vmodule = ""
+    # json = false
+    # backtrace = ""
+    # debug = true
+    # enable-block-tracking = false
+
+[p2p]
+    maxpeers = 200
+    port = 30303
+    # maxpendpeers = 50
+    # bind = "0.0.0.0"
+    # nodiscover = false
+    # nat = "any"
+    # netrestrict = ""
+    # nodekey = ""
+    # nodekeyhex = ""
+    # txarrivalwait = "500ms"
+    # [p2p.discovery]
+        # v4disc = true
+        # v5disc = false
+        # bootnodes = []
+        # bootnodesv4 = []
+        # bootnodesv5 = []
+        # static-nodes = []
+        # trusted-nodes = []
+        # dns = []
+
+# [heimdall]
+    # url = "http://localhost:1317"
+    # "bor.without" = false
+    # grpc-address = ""
+
+[txpool]
+    nolocals = true
+    pricelimit = 25000000000
+    accountslots = 16
+    globalslots = 32768
```

Comment:
realizing our use case is a bit specialized

perhaps we can work together on crafting a 'high-bandwidth' relayer PBSS config for the mesh project 

expect we'll need to tweak the following:
    accountslots = 64  # Increase slots to handle more transactions per account
    globalslots = 65536  # Increase global slots for overall transaction capacity
    
    ..along with some other values we'll likely need customized like bigger cache size , rpc setting, sync full , etc
    
curious your thoughts on having a dedicated config for that use case

Reply:
Yes I agree we should work together for a high-bandwidth config for the mesh project. The configs in this repo are for general use, you can always modify post install with ansible.  We can discuss further if you would like